### DISCUSSION TITLE
Voice output too freequent on twisty roads

### DISCUSSION THREAD

**Discussion by r4dh4l** [osmandapp/OsmAnd#19103] [Q&A]
Hi,

driving in a capital using Osmand `4.5.10` (F-Droid) the voice guidance has too much output. Imagine a road like this:

```
[north]

-|-
 |
 \
  |
 /
 |
 /
|
 \
 |

[south]
```

When I go from south to north and have to turn right on the crossroads in the north Osmand tells me something like this: "Turn slightly left on example road, turn slightly right on example road, turn slightly right on example road, turn slightly left on example road, turn right on example road 2."

Actually the only information I need is that I have to turn right on the "example road 2" at the crossroards. The information provided before are useless for me becaue there is no other possibility than driving ahead (there are no crossroads, just a twisty road).

Currently the amount of voice output is so disturbing that I don't use it at all.

Is there any possibility to reduce the output to a level that does not "describe" me the road I'm driving on but just tells me where I should go when there is more than one possibility?

**Comment by mwerle** [comment]
Possible duplicate of https://github.com/osmandapp/OsmAnd/discussions/18128?
